etc/ci/check_no_unwrap.sh

```diff
@@ -13,4 +13,8 @@ FILES=("components/compositing/compositor.rs"
        "components/compositing/pipeline.rs"
        "components/compositing/constellation.rs")
 
-! grep -n "unwrap(\|panic!(" "${FILES[@]}"
+# make sure the files exist
+ls -1 "${FILES[@]}"
+
+# make sure the files do not contain "unwrap" or "panic!"
+! grep -Hn "unwrap(\|panic!(" "${FILES[@]}"
```

Comment:
In that case, I think using long options would be better here. I've never seen -H used and had to look it up, and might as well spell out -n as well.